content/docs/architecture/resources/scheduledevent.md

```diff
@@ -1,12 +1,116 @@
 +++
 title = "ScheduledEvent"
+description = "A collection of Scenarios and Courses that are scheduled to run at a specific time."
 +++
 
-We are currently working on this...
+In HobbyFarm, a `ScheduledEvent` is a collection of [Scenarios](/docs/architecture/resources/scenario) and [Courses](/docs/architecture/resources/course) that are scheduled to run at a specific time. ScheduledEvents make use of an access code to restrict access to the event.
 
-TODO: Example
+Users must enter the access code to view and initiate the event. The access code is created during the creation of the ScheduledEvent and can be changed at any time. A ScheduledEvent can be configured to run on-demand or at a specific time.
```

Comment:
to start machines on-demand or at a specific time

Reply:
Updated.